cost-analyzer/templates/_helpers.tpl

```diff
@@ -96,14 +96,6 @@ Kubecost 2.0 preconditions
   {{- end }}
 {{- end -}}
 
-{{- define "federatedStorageCheck" -}}
```

Comment:
Based on the current plan, we will always have a store when using the statefulset in 3.0